designer/client/conditions/inline-conditions.js

```diff
@@ -77,6 +73,15 @@ class InlineConditions extends React.Component {
     }
   }
 
+  onClickSave = async () => {
+    const copy = this.props.data.clone()
+    const conditionResult = await InlineConditionHelpers.storeConditionIfNecessary(copy, this.state.conditions)
```

Comment:
export storeConditionIfNecessary, and import it with `import { storeConditionIfNecessary }` so you don't have to do the `InlineConditionHelpers.` It looks like a class currently

Reply:
Unfortunately I don't think I can, as soon as I do this the tests start failing which appears to be because sinon can't stub destructured imports https://sinonjs.org/how-to/stub-dependency/